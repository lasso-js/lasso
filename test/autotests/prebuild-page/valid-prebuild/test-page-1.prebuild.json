[
  {
    "slots": {
      "body": "<script src=\"/static/test-page-1/lasso$3.4.3/test/autotests/prebuild-page/valid-prebuild/a.js\"></script>"
    },
    "assets": [],
    "name": "test-page-1"
  }
]