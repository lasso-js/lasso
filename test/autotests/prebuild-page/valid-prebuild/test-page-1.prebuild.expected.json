[
  {
    "slots": {
      "body": [{
        "contentType": "js",
        "content": [
          {
            "inline": false,
            "code": {
              "src": "/static/test-page-1/lasso$4.0.1/test/autotests/prebuild-page/valid-prebuild/a.js"
            }
          }
        ]
      }]
    },
    "assets": [],
    "name": "test-page-1"
  }
]